app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -180,5 +177,19 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$scope', 'miqServ
     }
   };
 
+  var formOptions = function() {
+    API.get("/api/providers?expand=resources&attributes=id,name&filter[]=supports_block_storage?=true").then(function(data) {
```

Comment:
Does every StorageManager that supports `:block_storage` feature also supports creation of new cloud volumes? At the moment this seems to be true (since only OpenStack and Amazon support object storage), but will it always be true? When someone implements inventory (e.g. for Azure) it will appear on this list even if creating new volumes is not implemented yet.